contracts/wrapper/NameWrapper.sol

```diff
@@ -41,11 +39,15 @@ contract NameWrapper is
 
     bytes32 private constant ETH_NODE =
         0x93cdeb708b7545dc668eb9280176169d1c33cfd8ed6f04690a0bcc88a93fc4ae;
+    bytes32 private constant ETH_LABELHASH =
+        0x4f5b812789fc606be1b3b16908db13fc7a9adf7ca72641f84d75b47069d3d7f0;
     bytes32 private constant ROOT_NODE =
         0x0000000000000000000000000000000000000000000000000000000000000000;
 
     INameWrapperUpgrade public upgradeContract;
     uint64 private constant MAX_EXPIRY = type(uint64).max;
+    uint32 private constant PARENT_CONTROLLED_FUSES =
```

Comment:
I think this is probably clearer as just a constant.

Reply:
Will change